test/test-call-forwarding

```diff
@@ -12,7 +13,9 @@ def property_changed(property, value):
 def print_properties(cf):
 	properties = cf.GetProperties()
 
-	for p in properties:
+	for p in ["VoiceBusy", "VoiceNoReplyTimeout", "VoiceNotReachable",
+			"VoiceUnconditional", "VoiceNoReply",
+			"ForwardingFlagOnSim"]:
```

Comment:
Got it, but taking into account that that is Ubuntu Touch's test plan, I am not sure if upstream would accept this change if that is the only reason. If we want to fully automatize CallForwarding testing I think we should create a different script similar to the ones you created for testing the SIM interface, because I do not think that the script fully fits our testing needs.